src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -304,6 +304,12 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             var functionDetails = await WebFunctionsManager.GetFunctionMetadataResponse(listableFunctions, hostOptions, _hostNameProvider);
             result.Add("functions", new JArray(functionDetails.Select(p => JObject.FromObject(p))));
 
+            // Add host.json to the payload
+            if (_environment.IsKubernetesManagedHosting())
```

Comment:
Can we add a unit test specifically for Kubernetes scenario?

Reply:
Thank you for you advice! Clever. Done.